ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -207,7 +219,7 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
           color: thresholdLineColor,
           value:
             thresholds.mode === 'Percent'
-              ? convertPercentThreshold(step.value, graphData.timeSeries, yAxis.max, yAxis.min)
+              ? convertPercentThreshold(step.value, graphData.timeSeries, y_axis?.max, y_axis?.min)
```

Comment:
To fix param types, I changed to use min and max from panel spec here since they will always be a number (if defined)

Reply:
It took me a little looking at `convertPercentThreshold` to understand why you'd use `y_axis` here instead of `yAxis`. It might be worth putting a comment about it, because I could totally see someone seeing it in the future and thinking it was a mistake and trying to switch them back.